src/integrationTest/reproductions/issue-173-empty/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'eu.xenit.docker'
+}
+
+createDockerFile {
+    from("alpine:edge")
+}
+
+dockerBuild {
+    repositories = ['issue-173']
+    tags = autotag.legacyTags()
+}
+
```

Comment:
Waarom hebben we een issue-173-empty en een issue-173-array met exact hetzelfde in behalve de tags?